bin/run_multiple_panels.py

```diff
@@ -0,0 +1,443 @@
+            p.starmap(run_single_building, map_args)
+
+    my_print('All processes finished, exiting.')
+
+
+def run_single_building(input_path, der_type='ev', sim_type='circuit_pausing', tech1='EV', tech2='Water Heating', simulation_name='ochre', output_path=None):
+    # run individual building case
+    my_print(f'Running OCHRE for building {simulation_name} ({input_path})')
+    if not os.path.isabs(input_path):
+        input_path = os.path.join(os.getcwd(), input_path)
+        
+    dwelling_args = {
+        # Timing parameters      
+        'start_time': dt.datetime(2018, 1, 1, 0, 0),  # year, month, day, hour, minute
+        'time_res': dt.timedelta(minutes=15),         # time resolution of the simulation
+        'duration': dt.timedelta(days=365),             # duration of the simulation
+        'initialization_time': dt.timedelta(days=1),  # used to create realistic starting temperature
+
+        # Input parameters
+        'hpxml_file': os.path.join(input_path, 'Input Files', 'in.xml'),
+        'schedule_input_file': os.path.join(input_path, 'Input Files', 'schedules.csv'),
+        'weather_path': weather_path,
+    
+        # Output parameters
+        'output_path': input_path,
+        'output_to_parquet': False,              # saves time series files as parquet files (False saves as csv files)
+        'verbosity': 7,                         # verbosity of time series files (0-9)
+        
+        'seed': int(input_path[-3:]),
+        
+        'Equipment': {
+        },
+    }
+
+    # determine output path, default uses same as input path
+    if output_path is None:
+        output_path = input_path
+    os.makedirs(output_path, exist_ok=True)
+    
+    
+    # create OCHRE building based on der type
+    if der_type == 'ev':
+        dwelling = setup_ev_run(simulation_name, dwelling_args)
+    else: 
+        # baseline ResStock dwelling
+        dwelling = Dwelling(simulation_name,**dwelling_args)           
+                            
+    # run simulation
+    if sim_type == 'baseline':
+        # run without controls
+        dwelling.simulate()
+    
+    elif sim_type == 'circuit_sharing':
+        circuit_sharing_control(sim_type, dwelling, tech1, tech2, output_path)
+    
+    elif sim_type == 'circuit_pausing':
+        circuit_pausing_control(sim_type, input_path, dwelling, tech1, output_path)
+    
+    
+def setup_ev_run(simulation_name, dwelling_args):
+
+
+    equipment_args = {
+        'Electric Vehicle':{
+            # Equipment parameters
+            'vehicle_type': 'BEV',
+            'charging_level': 'Level 2',
+            "capacity": 57.5,
+        }
+    }
+
+    # Initialize equipment
+    # equipment = ElectricVehicle(**equipment_args)
+
+    dwelling_args['Equipment']=equipment_args
+    # print(dwelling_args)
+    dwelling = Dwelling(simulation_name, **dwelling_args)
+
+
+    return dwelling
+
+
+def circuit_sharing_control(sim_type, dwelling, tech1, tech2, output_path):   
```

Comment:
I'd love to split circuit sharing and circuit pausing into their own separate file in bin. (or @mnblonsky might have a better suggestions for where it should live). But this is absolutely something that's going to be useful for other projects and I'd like to get it into develop.